docs/_build/html/_sources/upgrade_skaffold.md.txt

```diff
@@ -0,0 +1,88 @@
+# From Skaffold
+## Before you begin
+* [Install Tilt](quickstart.html)
+* If you're new to Tilt try stepping through [a Simple Tiltfile](first_config.html) first.
+
+## Differences between Skaffold and Tilt
+* Skaffold streams the logs for services it started. Tilt provides a UI right in your console to view all of your running services and other relevant errors in addition to the log view that Skaffold provides.
```

Comment:
maybe say something like "We found one combined stream of all services difficult to use & understand for anything beyond the most simple apps." i.e., a justification for why Tilt does it differently